docs/self-hosting/README.md

```diff
@@ -61,10 +61,10 @@ Follow the steps below:
 
```

Comment:
dokcer-compose links above in the curl command (https://cdn-files.openblocks.dev/docker-compose.yml)